.github/workflows/publish-package-to-npm.yml

```diff
@@ -47,17 +47,29 @@ jobs:
     needs: publish
     runs-on: ubuntu-latest
     steps:
-      # Checkout to target branch
-      - uses: actions/checkout@v2
-
-      - name: Get package info
-        id: package
-        uses: codex-team/action-nodejs-package-info@v1
+        # Checkout to target branch
+        - uses: actions/checkout@v4
+  
+        - name: Get package info
+          id: package
+          uses: codex-team/action-nodejs-package-info@v1
 
-      - name: Send a message
-        uses: codex-team/action-codexbot-notify@v1
-        with:
-          webhook: ${{ secrets.CODEX_BOT_NOTIFY_EDITORJS_PUBLIC_CHAT }}
-          message: '📦 [${{ steps.package.outputs.name }}](${{ steps.package.outputs.npmjs-link }}) ${{ steps.package.outputs.version }} was published'
-          parse_mode: 'markdown'
-          disable_web_page_preview: true
+        - name: Set link to the github repository
+          if: ${{ github.event_name != 'release ' }}
```

Comment:
I think this `if` make sense only if it will be a common workflow. For now we have 2 different workflows: at editor.js repo and at tool's repo in https://github.com/editor-js